### PR TITLE
feat: add support for html 5 tags

### PR DESCRIPTION
**Pull request by BlakeStearman**
Adds support for the following HTML 5 tags with the `style` attribute:
* `article`
* `aside`
* `details`
* `footer`
* `header`
* `main`
* `mark`
* `nav`
* `section`
* `summary`
* `time`

Adds `pre`, `code`, and `blockquote` tags with `style` attribute.

Adds test case for sanitization of contents of `pre` tag.

Also adds the `aria-label` and `aria-hidden` attributes to the `div` tag.

### CODE REVIEW

File: src/index.test.ts [hunk 1]
```diff
@@ -50,13 +50,13 @@ describe("Sanitizer", () => {
 
     // Extending the defaults
     const sanitizer2 = new Sanitizer(
-      { allowCommentTag: false, whiteList: { blockquote: [] } },
+      { allowCommentTag: false, whiteList: { blink: [] } },
```
Comment (BlakeStearman) [inline src/index.test.ts:53]: The test for extending the defaults was changed since we added `blockquote` to the main whitelist.

**Review by dasa — APPROVED** [review]
LGTM, but description should include blockquote, code, and pre as well.

**Comment by BlakeStearman** [comment]
> LGTM, but description should include blockquote, code, and pre as well.

Good catch. Description has been updated.

**Comment by BlakeStearman** [comment]
@CodeBarker I did not include those, no. I can take another pass at addressing those specifically.

**Pull request merged**